packages/zendesk-adapter/src/change_validators/guide_theme_update_metadata.ts

```diff
@@ -0,0 +1,66 @@
+/*
+*                      Copyright 2024 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import {
+  ChangeValidator,
+  getChangeData,
+  isInstanceChange,
+  isModificationChange, isReferenceExpression,
+  SeverityLevel,
+} from '@salto-io/adapter-api'
+import _ from 'lodash'
+import { GUIDE_THEME_TYPE_NAME } from '../constants'
+
+const MANIFEST_FIELDS = ['author', 'name', 'version']
+/**
+ * this filters check that there has been no change to the theme metadata. to change the theme metadata the user needs
+ * to update the manifest file and not the nacl
```

Comment:
1. nits:
```suggestion

/**
 * this change validator checks that there has been no change to the theme metadata. to change the theme metadata the user needs
 * to update the manifest file and not the nacl
```
2. if these fields are also written in the manifest.json, is there a reason to have them in the nacl at all?
3. (related) when a user copies over both the manifest and the nacl changes from another env, won't they get these errors even though the change they made is valid?
4. seems we also have here "Moving a theme to a different brand is not supported" which is not listed?

Reply:
re 1 - we can still generate even if the name is hidden, no?

Reply:
opened a ticket to improve this CV https://salto-io.atlassian.net/browse/SALTO-5315